src/main/java/com/ericsson/eiffel/remrem/shared/MsgService.java

```diff
@@ -5,4 +5,6 @@
 
 public interface MsgService {
     String generateMsg(String msgType, JsonObject bodyJson);
+    String validateMsg(String msgType, JsonObject bodyJson);
```

Comment:
Since this is a public interface we should have good documentation. If you rebase on the 0.1.4 version you can see how to document.